docs/faq.md

```diff
@@ -42,28 +44,19 @@ and instructions to host a _listen server_:
 [basic-listen-server.md](hosting-a-server-with-northstar/basic-listen-server.md)
 {% endcontent-ref %}
 
-### Q: My main menu screen is blank or only cinematics show up!
-
-A: Please remove conflicting mods such as `better.serverbrowser` and reinstall _Northstar core mods_ (those that start with `Northstar.`, are in the [NorthstarMods repository](https://github.com/R2Northstar/NorthstarMods), and included in the release zip).\
-Try deleting `enabledmods.json` as well. Otherwise pay attention in console for your errors if you know what you're doing.
-
```

Comment:
Is there a reason this was removed from FAQ but not moved to Troubleshooting?

Reply:
I have forgot. Sorry. If you don't mind just copying it to the troubleshooting page cause I'm in school right now

Reply:
I don't know if i can do it on phone. Will try